components/AppBar/examples/AppBarImageryExample.swift

```diff
@@ -126,6 +126,7 @@ extension AppBarImagerySwiftExample {
       if cell == nil {
         cell = UITableViewCell(style: .default, reuseIdentifier: "cell")
       }
+      cell!.selectionStyle = UITableViewCellSelectionStyle.none
```

Comment:
Nits: You can do `cell!.selectionStyle = .none` to make it a little shorter. There's probably also a way to write it without the force unwrapping

Reply:
@andrewoverton Good to know! Addressed in 257e5e2c93e270b4720908a18ea04a569c19838a